bugs_and_glitches.md

```diff
@@ -527,12 +538,308 @@ HandleAICowardice:
 	...
 ```
 
-## Challenge host uses wrong name for the first rival
+### Phantom Venusaur will never be obtained through Card Pop
+
+([Video](https://youtu.be/vr8MZwW6gpI?si=qZuMBGRSrMyoGVJj&t=60))
+
+The name-checking routine used by Card Pop! to generate what card the player will receive is flawed, preventing one from ever receiving the Venusaur Phantom Card.
```

Comment:
My bad, I thought it must have been the root of the issue, especially since it's the only part of the card pop file to mention a bug.
Has a fix for "Unobtainable Phantom Venusaur" being made yet, or should I just replace that section with a "TODO"?

Reply:
There's a potential fix that is easy, yeah. Although it's arguable whether it's a bug or a design flaw (in pokecrystal those two are different pages, but in poketcg we don't have a page for design flaws).

Reply:
I left it as a todo in the mean time, while we wait for a consensus to be drawn.